test/parallel/test-readline-keys.js

```diff
@@ -136,7 +136,7 @@ addTest('a\x1baA\x1bA', [
 ]);
 
 // xterm/gnome ESC [ letter (with modifiers)
-/* eslint-disable max-len */
+
```

Comment:
I don’t think you want to be inserting blank lines for all the places you’re removing the `eslint-disable` comments. The `// xterm` comment above this is probably intended to describe the following line, which is less obvious when there’s a blank line between the comment and the code it’s annotating.

Reply:
Agreed. The extra blank line is how it is auto-fixed by ESLint, and they probably have their reasons, but I'll go through and remove them.

Reply:
```suggestion
```